src/foundation.cs

```diff
@@ -15356,4 +15356,10 @@ interface NSMeasurementFormatter : NSSecureCoding {
 		[Export ("stringFromUnit:")]
 		string ToString (NSUnit unit);
 	}
+
+	[iOS (6,0), Mac (10,8), Watch (2,0), TV (9,0)]
+	[BaseType (typeof (NSObject), Name = "NSXPCListenerEndpoint")]
+	interface NSXpcListenerEndpoint : NSSecureCoding
```

Comment:
Just a FYI: we already have a PR for this class (if slightly different: https://github.com/xamarin/xamarin-macios/pull/2315/files#diff-7ea102419ab3567d665e35d010e90a58R14440).

Please check if `[DisableDefaultCtor]` still applies in the current beta, otherwise this looks fine.

Reply:
Yes, is not explicitly set in headers but documentation agrees on this having a `[DisableDefaultCtor]` good catch!